.github/workflows/My-ci-pipeline.yml

```diff
@@ -0,0 +1,34 @@
+name: Continuous Integration
+on:
+  push:
+    paths-ignore:
+      - '**.md'
+      - '**.lock'
+      - '.github\**'
+      - '.idea\**'     
+      - '.pytest_cache\**'
+      - '.venv\**'
+      - '.vscode\**'
+
+  pull_request:
+    paths-ignore:
+      - '**.md'
+      - '**.lock'
```

Comment:
I'd be tempted to disagree with this one; if your `poetry.lock` file changes then our dependencies have updated, and it still feels worthwhile to get the validation that our tests are still passing even with new library versions

Reply:
updated by remoing lock item in this file